src/zenml/integrations/pytorch_lightning/materializers/pytorch_lightning_materializer.py

```diff
@@ -27,28 +29,29 @@
 class PyTorchLightningMaterializer(BaseMaterializer):
     """Materializer to read/write PyTorch models."""
 
-    ASSOCIATED_TYPES = (Trainer,)
+    ASSOCIATED_TYPES = (Module,)
```

Comment:
This is a breaking change, isn't it ? All existing steps that still return `Trainer` will break now. Is this something you would like to avoid by keeping the old materializer around ? Might also be fine to just drop it, given that it doesn't look like it was working to begin with.

This also makes me wonder if our example doesn't also need fixing.

Reply:
Oh, wait, we don't have a pytorch lighting example.

Reply:
We don't actually have an example for it, that's why I don't it has been ever used. For that I don't think we need to keep the old one I guess